parse_issues.py

```diff
@@ -0,0 +1,76 @@
+import requests
```

Comment:
not sure if there is a convention formally .. but usually we have just some global variables in the global scope of the module and the code actually we have inside functions .. maybe a `main` function and you can call it at the end of the file, as mentioned before.